libraries/base/Data/OldList.hs

```diff
@@ -241,9 +241,9 @@ infix 5 \\ -- comment to fool cpp: https://downloads.haskell.org/~ghc/latest/doc
 dropWhileEnd :: (a -> Bool) -> [a] -> [a]
 dropWhileEnd p = foldr (\x xs -> if p x && null xs then [] else x : xs) []
 
--- | The 'stripPrefix' function drops the given prefix from a list.
--- It returns 'Nothing' if the list did not start with the prefix
--- given, or 'Just' the list after the prefix, if it does.
+-- | /O(min(m,n))/. The 'stripPrefix' function drops the given prefix from a
```

Comment:
Let's save this for another patch and remain consistent with the other uses in this module.

Reply:
Hey @hvr ,

Thanks for pointing this out!

Looks like there are currently (more or less, the regex isn't very precise) 31 occurrences of *italic*  Big-Os in `base`.

```bash
➜  ghc git:(master) grep -r -E "/O\(" libraries/base | wc -l
      31
```

Should I open a ticket to replace them all, so that Big-Os in `base` are consistently typeset?

Best regards,

Sven

Reply:
@supersven, that would be great!

Reply:
Cool :smile: 

I created: https://ghc.haskell.org/trac/ghc/ticket/16090